src/main/java/com/ashkan/moviesapi/services/implementations/ActorServiceImpl.java

```diff
@@ -25,18 +25,14 @@ public ActorServiceImpl(ActorRepository actorRepository) {
     }
 
     @Override
-    public Actor findById(int theId) {
-        Optional<Actor> result = actorRepository.findById(theId);
-
-        Actor actor = null;
-
+    public Actor findById(int id) {
+        Optional<Actor> result = actorRepository.findById(id);
+        Actor actor;
         if (result.isPresent()) {
             actor = result.get();
+        } else {
+            throw new RuntimeException("Did not find the actor- " + id);
```

Comment:
Take into account this exception should return a 404 Not Found code response, I'm not sure if the RuntimeException will generate that type of response.

Reply:
done.